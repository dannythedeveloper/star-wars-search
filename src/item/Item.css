.item {
    display: flex;
    flex-direction: column;
    text-align: center;
    border: gray solid 1px;
}