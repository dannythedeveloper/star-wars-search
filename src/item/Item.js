import React from 'react';
import './Item.css';

export default function Item(props) {
    return (
        <div className='item'>
            <h2>{props.name ? props.name : props.title}</h2>
        </div>
    )
}