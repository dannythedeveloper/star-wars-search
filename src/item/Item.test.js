import React from 'react';
import ReactDOM from 'react-dom';
import Item from './Item';
import renderer from 'react-test-renderer';

describe('Item Component', () => {
    it('render without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Item />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI', () => {
        const tree = renderer.create(<Item />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});