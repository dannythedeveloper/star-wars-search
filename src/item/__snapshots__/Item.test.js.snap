// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item Component renders the UI 1`] = `
<div
  className="item"
>
  <h2 />
</div>
`;
